recipe/meta.yaml

```diff
@@ -18,28 +17,39 @@ source:
     # into various problems around ABIs of numpy/C++ etc.
     # Hopefully, we can provide more specific builds per CPU architecture
     # in the future, see conda-forge/conda-forge.github.io#1261
-    - patches/0002-silence-log-that-tells-people-to-install-from-pip.patch
+    - patches/0001-silence-log-that-tells-people-to-install-from-pip.patch
     # backport nmslib/nmslib#488 for compatibility with newer pybind11
```

Comment:
I think this comment is not needed anymore

Reply:
indeed

Reply:
I will remove it upon merging. Don't want to run the prefect just to rebuild without this comment.